src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3768,7 +3768,7 @@ bool os::protect_memory(char* addr, size_t bytes, ProtType prot,
   // memory, not a big deal anyway, as bytes less or equal than 64K
   if (!is_committed) {
     commit_memory_or_exit(addr, bytes, prot == MEM_PROT_RWX,
-                          "cannot commit protection page");
+                          "cannot commit protection page", mtNone);
```

Comment:
This should probably be something else than mtNone.

Reply:
Changed to `mtInternal`. When `protect_memory` is called with uncommitted area (`is_committed == false`), the flag is mtInternal.